metricflow/model/data_warehouse_model_validator.py

```diff
@@ -539,7 +538,7 @@ def run_tasks(
         return SemanticManifestValidationResults.from_issues_sequence(issues)
 
     def validate_semantic_models(
-        self, model: PydanticSemanticModel, timeout: Optional[int] = None
+        self, model: PydanticSemanticManifest, timeout: Optional[int] = None
```

Comment:
Huh. Should we need to rename the variables from model to manifest here?

Also, the typechecking failure here is disconcerting.